Makefile

```diff
@@ -45,6 +34,10 @@ CT_COVER_EXPORT_PREFIX ?= $(PROFILE)
 
 export REBAR_GIT_CLONE_OPTIONS += --depth=1
 
+.PHONY: nothing
```

Comment:
what is this? make nothing?

Reply:
for testing purpose.

Reply:
:scissors: ?

Reply:
I can do `time make nothing` to see how much speedup it is .